tests/foreman/ui/test_template.py

```diff
@@ -131,10 +131,10 @@ def test_negative_create_without_type(self):
                     custom_really=True,
                     template_type='',
                 )
-                self.assertEqual(
-                    context.exception.message,
-                    'Could not create template "{0}" without type'.format(name)
-                )
+            self.assertEqual(
+                context.exception.message,
+                'Could not create template "{0}" without type'.format(name)
```

Comment:
prefix the formatting string with u name may contain unicode characters and that will fail on Python 2.